packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -486,6 +486,8 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
     },
     reporting: {
       cloudMinimumRequirements: `${KIBANA_DOCS}reporting-getting-started.html#reporting-on-cloud-resource-requirements`,
+      browserSystemDependencies: `${KIBANA_DOCS}reporting-troubleshooting.html#reporting-troubleshooting-system-dependencies`,
```

Comment:
That anchor doesn't exist, do you perhaps mean this one instead?
```suggestion
      browserSystemDependencies: `${KIBANA_DOCS}reporting-troubleshooting.html#reporting-troubleshooting-system-requirements`,
```

Reply:
Thank you @lcawl for explaining the docs build failure. :) The link that I wanted to point to is here: https://www.elastic.co/guide/en/kibana/current/secure-reporting.html#install-reporting-packages

So I changed the code to fix that in this commit: 3fabcb2adbd